be/src/olap/delta_writer.cpp

```diff
@@ -46,12 +46,20 @@ DeltaWriter::~DeltaWriter() {
 }
 
 void DeltaWriter::_garbage_collection() {
-    TxnManager::instance()->rollback_txn(_req.partition_id, _req.txn_id,_req.tablet_id, _req.schema_hash);
-    StorageEngine::instance()->add_unused_rowset(_cur_rowset);
+    OLAPStatus rollback_status = TxnManager::instance()->rollback_txn(_req.partition_id, 
```

Comment:
In this place, if data has generated but rowset is not been created, how to GC data?